interface/forms/eye_mag/php/taskman_functions.php

```diff
@@ -340,10 +356,15 @@ function make_document($task)
         'use_kwt' => true,
         'keep_table_proportions' => true
     );
+
     $pdf = new mPDF($config_mpdf);
     if ($_SESSION['language_direction'] == 'rtl') {
         $pdf->SetDirectionality('rtl');
     }
+    $pdf->shrink_tables_to_fit = 1;
+    $keep_table_proportions = true;
+    $pdf->use_kwt = true;
```

Comment:
Note the above 3 lines are not needed. mPDF version was upgrades and note these are included in the $config_mpdf code block above.